src/sections/PageHero.scss

```diff
@@ -0,0 +1,149 @@
+.sectionContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: linear-gradient(90deg, #fdcf0c, #f89b21);
+  width: 100%;
+}
+
```

Comment:
Please put these components/scss files in their own folder. For example, the path would look like src/sections/PageHero/ for the PathHero files